### ISSUE TITLE
RuntimeError: Unsupported database scheme

### ISSUE THREAD

**Issue by geosphere** [tulir/mautrix-facebook#104]
Hi! When I upgrade the container to latest state and restart I get this error:  

```
INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.

INFO  [alembic.runtime.migration] Will assume transactional DDL.

INFO  [alembic.runtime.migration] Running upgrade 2162f7ae5365 -> 3c59aa2a3af9, Add messenger domain to db

INFO  [alembic.runtime.migration] Running upgrade 3c59aa2a3af9 -> 3d4f3a252006, Add double puppet base URL to puppet table

[2020-10-17 01:40:49,132] [INFO@mau.init] Initializing mautrix-facebook 0.1.0+dev.d530f4f0

[2020-10-17 01:40:49,187] [CRITICAL@mau.init] Unexpected error in initialization

Traceback (most recent call last):

  File "/usr/lib/python3.8/site-packages/mautrix/util/program.py", line 93, in _prepare

    self.prepare()

  File "/usr/lib/python3.8/site-packages/mautrix/bridge/bridge.py", line 81, in prepare

    self.prepare_bridge()

  File "/usr/lib/python3.8/site-packages/mautrix_facebook/__main__.py", line 53, in prepare_bridge

    self.matrix = context.mx = MatrixHandler(context)

  File "/usr/lib/python3.8/site-packages/mautrix_facebook/matrix.py", line 41, in __init__

    super().__init__(bridge=context.bridge)

  File "/usr/lib/python3.8/site-packages/mautrix/bridge/matrix.py", line 64, in __init__

    self.e2ee = EncryptionManager(

  File "/usr/lib/python3.8/site-packages/mautrix/bridge/e2ee.py", line 79, in __init__

    raise RuntimeError("Unsupported database scheme")

RuntimeError: Unsupported database scheme

```

**Comment by tulir** [comment]
So use a supported database scheme?

**Comment by geosphere** [comment]
Thank you for your response. What has changed at database scheme over the last few months? Am I missing any migrations?

**Comment by tulir** [comment]
If the problem was using `postgresql://` instead of `postgres://`, this will be fixed in mautrix-python v0.8.9. Other database schemes aren't supported anymore.
